test/hotspot/jtreg/serviceability/dcmd/vm/SystemDumpMapTest.java

```diff
@@ -28,6 +28,7 @@
 import jdk.test.lib.process.OutputAnalyzer;
 
 import java.io.*;
+import java.lang.StringBuilder;
```

Comment:
Nit: `java.lang.*` are imported by default. I don't see it used, so maybe a left over?

Reply:
Yep, a bunch of those are not needed anymore.